pkgs/top-level/stage.nix

```diff
@@ -51,9 +79,23 @@ let
     };
 
   stdenvBootstappingAndPlatforms = self: super: {
-    stdenv = stdenv // { inherit platform; };
-    inherit buildPackages
-      system platform crossSystem;
+    inherit stdenv buildPackages
+      buildPlatform hostPlatform targetPlatform;
+  };
+
+  # The old identifiers for cross-compiling. These should eventually be removed,
```

Comment:
Oh I already removed crossSystem (it's just used for picking stdenvs now). I was going to remove `stdenv.cross` after in a kind of victory lap :).